pyproject.toml

```diff
@@ -33,6 +33,11 @@ ollama = "^0.2.0"
 openai = "^1.29.0"
 uvloop = "^0.19.0"
 httptools = "^0.6.1"
+anthropic = "^0.26.1"
+google-generativeai = "^0.5.4"
+mistralai = "^0.2.0"
+pytest = "^8.2.1"
+mistral = "^18.0.1"
```

Comment:
Why the dupplicated dep? we only need https://github.com/mistralai/client-python right?